src/EnergyPlus/PluginManager.cc

```diff
@@ -105,13 +105,18 @@ namespace PluginManagement {
     PyObjectWrap (*EP_PyObject_CallFunction)(PyObjectWrap, const char *) = nullptr;
     PyObjectWrap (*EP_PyObject_CallFunction3Args)(PyObjectWrap, const char *, PyObjectWrap, PyObjectWrap, PyObjectWrap) = nullptr;
     PyObjectWrap (*EP_PyObject_CallMethod)(PyObjectWrap, const char *, const char *) = nullptr;
+    PyObjectWrap (*EP_PyObject_CallMethod1Arg)(PyObjectWrap, const char *, const char *, PyObjectWrap) = nullptr;
+    PyObjectWrap (*EP_PyObject_CallMethod2ObjArg)(PyObjectWrap, PyObjectWrap, PyObjectWrap, PyObjectWrap) = nullptr;
     int (*EP_PyList_Check)(PyObjectWrap) = nullptr;
     unsigned long (*EP_PyList_Size)(PyObjectWrap) = nullptr;
     PyObjectWrap (*EP_PyList_GetItem)(PyObjectWrap, size_t) = nullptr;
     int (*EP_PyUnicode_Check)(PyObjectWrap) = nullptr;
     int (*EP_PyLong_Check)(PyObjectWrap) = nullptr;
     long (*EP_PyLong_AsLong)(PyObjectWrap) = nullptr;
     void (*EP_Py_SetPythonHome)(const wchar_t *) = nullptr;
+    PyObjectWrap (*EP_Py_BuildValue)(const char *) = nullptr;
+    PyObjectWrap (*EP_PyLong_FromVoidPtr)(void *) = nullptr;
+    PyObjectWrap (*EP_PyString_FromString)(const char *) = nullptr;
```

Comment:
We are now passing a pointer to the current `state` instance into the plugin override functions, so that those functions can then pass that state around as well.  Needed to add a few new functions to call into the Python DLL appropriately.